wpeverest/user-registration
# UR-1711 Bug - Moved membership settings logic to membership

### All Submissions:






### Changes proposed in this Pull Request:
Moved the code to add membership settings from pro to membership addon.



Closes # .

### How to test the changes in this Pull Request:

1. Activate Pro -> activate membership -> deactivate Pro
2. No error must appear in log related to settings required.

### Types of changes:




### Other information:




### Changelog entry

> Moved membership settings logic to membership.
